packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -760,6 +760,33 @@ QUnit.module('Menu tests', {
         assert.notOk(submenu.option('visible'), 'submenu was closed');
     });
 
+    [true, false].forEach(hideSubmenuOnFocusOut => {
+        QUnit.test(`Close submenu when element lose focus and _hideSubmenuOnFocusOut=${hideSubmenuOnFocusOut}`, function(assert) {
```

Comment:
let's rename test, now sounds like menu should be closed in both scenarios

Reply:
Thanks, I have renamed the test: [69ee2668a0da82519efd21e28f1b049f847a0cec].